screens/CategoryScreens/EditCategoryScreen.tsx

```diff
@@ -25,15 +24,15 @@ export default function EditCategoryScreen({ navigation, route }: RootStackScree
   const [confirmDelete, setConfirmDelete] = useState(false)
   const [newDetails, setNewDetails] = useState(details)
 
-  const passwordHash = useAuth();
-
-  useUnauthRedirect();
+  const passwordHash = useAuth({
+    onRetrieved: (passwordHash) => getCategories({ variables: { passwordHash } }),
```

Comment:
how are you able to use a variable "getCategories" before it is defined??

Reply:
Hoisting